.github/workflows/pull-request-test.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v4
-      - name: Set up JDK 11
+      - name: Set up JDK 17
         uses: actions/setup-java@v4
         with:
-          java-version: '11'
+          java-version: '17'
```

Comment:
To see if new changes break with Java 11, what about also running with 11 without the robolectric tests?

Reply:
I added Java 11 back in https://github.com/amplitude/Amplitude-Kotlin/pull/200/commits/467c51068ade551dac725db19100ab6c7d247a4b. This should run the build with JDK 11 as well.